src/util/IfOverflow.ts

```diff
@@ -0,0 +1 @@
+export type IfOverflow = 'hidden' | 'visible' | 'discard' | 'extendDomain';
```

Comment:
this file is now just a singular type. Maybe we move it somewhere else? I just left it for now

Reply:
No harm in singular type files imgo